### PR TITLE
Portfolio Contact Form

### PR DESCRIPTION
**Pull request by Wolfmikerson**
This pull request includes:

1- Added Contact Form to my portfolio

2- Collect the form data by using the Formspree service.

3- Remote Pair Programming contribution

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,148 +1,181 @@
...
+      project that needs coding. I’d love to help with it</p>
+    <form class="form default-display-flex" action="https://formspree.io/f/xbjqpynb" method="post">
+      <input class="form-element" type="text" id="name" name="user_name" placeholder="Wolfmikerson" maxlength="30"
+        required>
+      <input class="form-element" type="email" id="mail" name="user_email" placeholder="wolfmikerson@gmail.com"
+        required>
+      <textarea class="form-element" id="msg" name="user_message" placeholder="Write your message here" maxlength="500"
+        required></textarea>
+      <button class="but but1 form-element" type="submit">get in touch</button>
+    </form>
```
Comment (barackm) [inline index.html]: - [ ] It seems that there are some issues with your form, it does not look like the original one in Figma yet, see the difference below 👇 
- Your form
image

- Figma design
image

Can you see the difference? please let us fix the issue to make the form look like the Figma design 😉 

- [ ] Kindly let us add Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template. 👍

**Review by barackm — CHANGES_REQUESTED** [review]
Hi @Wolfmikerson, 👋 

You have done an amazing job so far! 👏👏👏
However, there are some issues that you still need to work on to go to the next project. But remember you are almost there 😉  😉 !

### To highlight
✔️  Descriptive Pull request title
✔️  Descriptive Pull request summary
✔️  Descriptive commit messages 👏 

# STATUS: Required Changes ♻️
### _Check the comments under the review._ 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 2]
```diff
@@ -349,3 +355,61 @@ body {
...
+
+input.form-element {
+  height: 48px;
+  width: 327px;
+}
+
+textarea.form-element {
+  height: 170px;
+}
+
```
Comment (safafa) [inline css/style.css:414]: - [ ] As the previous reviewer requested. Kindly add all the buttons interactions such as hover, active.  as shown in the figma template. 
image
Following the original design is one of the project requirements. As it's an important skill for front-end development.

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @Wolfmikerson  @MrRamoun , Good job working on some of the changes requested by the previous reviewer 👏🏼 👏🏼 

But,  there are still some which aren't addressed yet.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: css/style.css (hunk 2 above)
Comment (carlos-ssh) [reply css/style.css:414]: Kindly follow the requested requirements and provide the correct colors and styles.

**Review by carlos-ssh — APPROVED** [review]
Hi @Wolfmikerson , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**